app/models/service/containers.rb

```diff
@@ -1,6 +1,10 @@
 module Service
   class Containers
-    def self.start_container!(wizard_state)
+    def errors
```

Comment:
I'm wondering if errors should be here or in the DockerContainerWizardState itself @witlessbird ?